software/apache/README.md

```diff
@@ -25,171 +21,144 @@ This guide walks you through setting up Apache natively on your MacOS with the a
-### Vhosts and the Hosts File
+    /usr/local/etc/php/5.6/php.ini
+    /usr/local/etc/php/7.0/php.ini
 
-Using Apache 2.4  you should be able to add vhosts using the `vhost` directory.
+## Setting up a new PHP site in Apache
 
-In your local apache `/usr/local/etc/httpd/` directory run the following commands.
+### Create the vHost file
+Each site will have it's own _vhost_ file in `/usr/local/etc/httpd/vhosts`.
 
-    $ mkdir vhosts
-    $ cd vhosts
-    $ touch <your_project_name>.conf
+:warning: You’ll need to swap out the following placeholders with your own settings:
 
-Open the file `<your_project_name>.conf`
+  * `<project_name>`: Domain safe name such as `seesparkbox`
+  * `<project_root>`: Web root such as `/projects/seesparkbox/static/dist`
+  * `<php_port>`: Port to the PHP-FPM instance, such as `9071` for 7.1
 
-Copy and paste the following vhost configuration inside that file:
+Copy, paste, & edit the following vhost configuration inside the
+file `/usr/local/etc/httpd/vhosts/<project_name>.conf`:
 
-    <VirtualHost 127.0.0.1:80>
-      ServerAdmin youremail@heysparkbox.com
-      DocumentRoot "<your_path_to_files>/<your_project_name>/web"
-      ServerName <your_project_name>.local
+    <VirtualHost *:80>
+        ServerName <project_name>.local
+        DocumentRoot "<project_root>"
+        ProxyPassMatch ^/(.*\.php(/.*)?)$ fcgi://127.0.0.1:<php_port><project_root>/$1
+        ErrorLog "/usr/local/var/log/httpd/<project_name>-error_log"
+        CustomLog "/usr/local/var/log/httpd/<project_name>-access_log" common
+        DirectoryIndex index.php index.html
 
-      <Directory "<your_path_to_files>/<your_project_name>/web">
-        Options Indexes MultiViews FollowSymLinks
-        AllowOverride All
-        Require all granted
-        DirectoryIndex index.php
-      </Directory>
+        <Directory "<project_root>">
+          AllowOverride All
+          DirectoryIndex index.php index.html
+          Options Indexes MultiViews FollowSymLinks
+          Require all granted
+       </Directory>
     </VirtualHost>
 
-You’ll need to swap out the bolded text with your own settings.
-
 Save this file, and restart apache.
 
-Host file, open with `sudo` to make changes to /etc/hosts
-Add the following to the end of the file.
+### Add the site to your `hosts` file
+With `sudo` open `/etc/hosts` and add the following to the end of the file:
 
-    127.0.0.1 <your_project_name>.local
+    127.0.0.1 <project_name>.local
 
 ## Installing Composer
 
-We’ll be installing composer globally. 
+Installing composer globally.
 
-    $ cd ~
-    $ wget https://raw.githubusercontent.com/composer/getcomposer.org/1b137f8bf6db3e79a38a5bc45324414a6b1f9df2/web/installer -O - -q | php -- --quiet
-    $ mv composer.phar /usr/local/bin/composer
+    cd ~
+    wget https://raw.githubusercontent.com/composer/getcomposer.org/1b137f8bf6db3e79a38a5bc45324414a6b1f9df2/web/installer -O - -q | php -- --quiet
+    mv composer.phar /usr/local/bin/composer
```

Comment:
Any reason not to simplify this by using Homebrew to install?: `brew install composer`

Reply:
I wavered on this one. The official composer instructions are where this came from, but I'd be ok with going brew 👍